packages/v4/src/content/design-guidelines/components/tooltip/tooltip.md

```diff
@@ -2,51 +2,44 @@
 id: Tooltip
 section: components
 ---
-A **tooltip** provides short, clarifying descriptions of elements on a page. They are typically used to clarify the meaning of icons.
+A **tooltip** is in-app messaging used to identify elements on a page with short, clarifying text.
 
-## Usage
+For information on other forms of on-screen help, see [Popover](/components/popover/design-guidelines) and [Hint](/components/hint/design-guidelines). 
 
-<img src="./img/tooltips.png" alt="Example of tooltip in a table"  width="990"/>
+## Usage 
+Use tooltips to:
+- Provide labels for unlabeled icons.   
 
-Use tooltips to define new or unfamiliar UI elements that aren’t described directly in the user interface, or to get additional data from a data point or element in a chart or table.
+<img src="./img/kebab-tooltip.png" alt="kebab icon with a “More options” tooltip label" width="582"/>  
 
-**Guidelines**
-* Keep your tooltips clear and concise. Use the fewest number of words you can without sacrificing meaning.
-* If the tooltip is a full sentence, end it with a period.
-* If information is needed for a user to complete a task (like a password character requirement), don’t hide it in a tooltip. Display it on the page instead.
-* Tooltips should provide new and valuable information. Never use a tooltip to repeat information already available in the UI.
-* Don’t use tooltips with question-circle icons to present contextual information in forms and other areas. Instead, use a popover.
+- Provide additional information on a data point or element in a chart or table.   
 
-**Common use cases**
-* On icons
-* In charts
+<img src="./img/bar-chart.png" alt='bar chart with a tooltip for Asia Pacific' width='306'/> 
```

Comment:
All of these images are really small haha

Reply:
Updated 2 of the images. I pinged you about the chart one.

Reply:
Changed chart image sizing to match the artboard